classes/checkout-gateway.php

```diff
@@ -14,7 +14,7 @@
 use WC_Order;
 use WC_Payment_Gateway;
 use Exception;
-use Zaver_Checkout_Settings;
+use Settings;
```

Comment:
This use statement will need to be updated when you add the namespace to the file.